python/tests/scripts/test_update_storage.py

```diff
@@ -110,6 +112,14 @@ def test_upgrade_script_s3_rbac_ok(moto_s3_endpoint_and_credentials, monkeypatch
     assert s3_storage.credential_name == USE_AWS_CRED_PROVIDERS_TOKEN
     assert s3_storage.credential_key == USE_AWS_CRED_PROVIDERS_TOKEN
 
+    expected = pd.DataFrame({"col1": [1, 2, 3], "col2": [4, 5, 6]})
```

Comment:
Testing that we can create and access a library without `AWS_ACCESS_KEY_ID` and `AWS_ACCESS_KEY_ID` being explicitly given feels like it should be asserted in a dedicated test, rather than a test that asserts correctness of a script. 

Should we move this assertion to a simple, dedicated test?

Reply:
Agreed. I was torn between whether putting the new lines in an existing test or having a new test earlier. 
Fix pushed